### DISCUSSION TITLE
2025-01-01 Weekly Meeting 192

### DISCUSSION THREAD

**Discussion by indiebio** [omigroup/omigroup#508] [Weekly Meetings]
Weekly Meeting 192 is happening on Wednesday, 1 January 2025 from 9:00 AM PDT (17:00 UTC) - 10:00 AM PDT (18:00 UTC)

Hope everyone has a great new year!

You can add it to your calendar [here](https://calendar.google.com/calendar/u/1?cid=Y18wZHB1Z2Y5ZjgzZXE0cWVrbWI2b21xYmptZ0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t).

We'll be meeting in the omi-weekly-meeting channel of the [OMI Discord](https://discord.gg/jVaUteJyh7).
We regularly use the # omi-meeting-chat channel for text chat.

# Agenda
check in, shootin' the breeze (add agenda points or notes as comments). Informal format because many parts celebrate New Year today.

Specific talking point: OMI 2024 report and plans for 2025

**Comment by indiebio** [comment]
I would like to have a blogpost about what we achieved in 2024 (rather late than never) and what our goals are for 2025. Please add your comments.

Highlights: (please add yours)
- interesting conversations on Discord
- Interesting conversations documented on GitHub conversations
-- Compiling standards in nano-components https://github.com/omigroup/omigroup/discussions/504

Working groups: (please give a summary sentence and relevant links)
- MSF: jin's AI summary work
- glTF: specs approved, PR's merged, tensions navigated - lead by aaron, work by aaron, Yank Scally, add other names.
- media: growth of followers on social media channels. List them, list number of followers? List conversations had
- Virtual Worlds Museum - progress, award, mature to own Discord server
- Composable metaverse - conversation on Spaces, active dialogue on Discord channel
- highlights from omi-experiments

OMI members: (please add whatever you did that you are proud of)
- Solo launched Memotar
- Academic project proposal submitted - indiebio and knev
- iFire did stuff
- Sean RP1 progress
- 

Goals for 2025: (please suggest more)
- continue to improve our brand strategy, with the aim to attract contributing members. Focus on authentic, organic growth, quality over quantity. (lead by media group)
- continue to improve our reputation, to be a trusted source of specs, information and a diverse, healthy network of knowledge and people. (lead by media, glTF groups)
- develop a cross-platform conversational strategy (moving away from Spaces, looking into Youtube + Twitch, for example) (lead by media group)
- explore new ways to support sphttps://github.com/omigroup/omigroup/discussions/508ecs through nano-components (lead by glTF group)
- explore the social interoperability - how to work with different viewpoints, support plurality while maintaining accountability. (general, technically supported by MSF group)
- better integration with our w3c group. (lead by media group)
- technical experiment of portaling / interoperability through different virtual worlds (and wider) by hosting instances on different virtual worlds, along with a "kit" scene glTF file. Start with Hubs. (link to conversation)

**Comment by indiebio** [comment]
For ease of reading and commenting splitting these into comments:
Highlights: (please add yours)
- interesting conversations on Discord

**Reply by indiebio** [reply]
-- Portalling as a service https://github.com/omigroup/omigroup/discussions/506#discussioncomment-11711206

**Comment by indiebio** [comment]
- Interesting conversations documented on GitHub conversations

**Reply by indiebio** [reply]
-- Compiling standards in nano-components https://github.com/omigroup/omigroup/discussions/504

**Comment by indiebio** [comment]
- Working groups:

**Reply by indiebio** [reply]
media: growth of followers on social media channels. List them, list number of followers? List conversations had
-- https://omigroup.org/openbrush-where-its-been-and-where-its-going/
-- https://omigroup.org/notes-from-gltf-interactivity-extension/
-- we are on bluesky: @openmetaverse.bsky.social (fresh start, 10 followers so far)
-- [3,313 Followers](https://x.com/open_metaverse/verified_followers) on Spaces

**Reply by indiebio** [reply]
Composable metaverse: https://discord.com/channels/887789163045335081/1294511978265710703

**Comment by indiebio** [comment]
OMI members

**Reply by indiebio** [reply]
jin: https://x.com/ai16zdao/status/1874223078403682672
- ElizaOS: 7700+ stars, 2100+ forks, 223 contributors, 1.2k followers
  - Merged 850 PRs—Eliza is now the most multichain AI agent framework ever
- Launched AI Dev School: [http://eliza.gg/school](https://t.co/VGMTGcJ10z)
- 87 submissions made with ElizaOS for the Solana AI hackathon powered by 
[@sendaifun](https://x.com/sendaifun)

- Hundreds of teams are building on ElizaOS
- Eliza Labs x Stanford Future Digital Currency Initiative
- ai16z: First Solana AI memecoin to touch $2B market cap
- AI mindshare is over 50% of all crypto narratives
  - Solana had 50% of crypto users last month
- ai16z AUM now exceeds $30M, achieving 376x growth in 9 weeks
- Community growth:
  - 120k+ followers across 
[@ai16zdao](https://x.com/ai16zdao)
, 
[@pmairca](https://x.com/pmairca)
, and 
[@degenspartanai](https://x.com/degenspartanai)
 X accounts
  - 62k+ holders for ai16z, ~14k for degenai

2025 Outlook

- Eliza V2 Development: Improved architecture, scalability, and plugin management
- Developer Experience: Tailored AI agents, agentic GitHub workflows, and code assistants for faster onboarding
- Autonomous Investors: Launch V2 of autonomous investor agents (like degenai)
- Tokenomics:
  - Launch Eliza agent platform + dev/partnership programs
  - Launch RGPF program to strengthen open-source feedback loops
- Expanding Impact:
  - Pill web2 audiences on ElizaOS and DeFi
  - Build stronger developer pipelines and partnerships to accelerate adoption

**Reply by indiebio** [reply]
track instances of Virtual Worlds Museum instances - https://github.com/KEYFRAME-SEHU/mapping
Focus on VR and apps for now, with Horizon World ready: https://horizon.meta.com/world/478207822044529/?locale=en_US
OPEN SOURCE: https://www.virtualworlds.museum/web-xr-exhibit/open-source

**Reply by indiebio** [reply]
Pablo numinia: highlights: incentive systems, positioning, people who were building the last three market years, it is now starting to gain momentum, get second versions. Our vision has proved right, the value of open source and no-cost for land in the metaverse. Excited to see what's coming! Also hyperfy is exciting:  https://x.com/hyperfy_io/status/1876054095074881555
Agents are blowing up.
Goal for 2025: Getting Numinia Games to a viable organisation.
Goal for OMI in 2025? Having more metaversey events. e.g. in hyperfy, substrata, open source platforms

**Reply by indiebio** [reply]
Sean RP1: Highlight: 2024 was a building year to get core tech ready. We're getting close to a world demo.
Goal for 2025: Sustained ecosystem for metaverse. Removing friction in the XR industry. Build a browser using open standards.
subgoal: create a founding hall - [context link to follow](https://rp1.com/challenges)
Goal for OMI? File formats are good. Share our direction, get more feedback, work more on how we get interoperability really working. The biggest challenge of the industry is friction. Intranets with bridges is not the way to go! How do we truly bring together an interoperable XR experience. It takes scope, and a way of thinking differently. 
(what can we learn from the early days of the internet)

**Reply by indiebio** [reply]
Solo: highlight: launched memotar beta
Goal 2025: Year of agents? People learning how agents and XR work hand in hand. My vision is coming true! :) Getting more people on board with mixed reality agents.
Goal for OMI? More of the same. Efforts in glTF has been positive. Keep going! Standards, formats

**Reply by indiebio** [reply]
MTV: Goal for 2025: Live music streaming into virtual reality experiences

**Reply by indiebio** [reply]
Mrmetaverse: we survived another year! Coded a lot. 

OMI in 2025? Build greater cohesiveness. Create opportunities for each other. Find new flavours of success for ourselves. Celebrate each other.

**Reply by indiebio** [reply]
jimmy: highlight: Spaces campaign was good. Great to have those conversations. Have been keeping up with research in the AI space.  The intersection of AI and world models, multimodal, robotics ... all the things. Convergences are expected, in WebXR.
Even things not designed for immersive experiences are using game engine technologies, this is awesome. 3D is coming into the foreground now. And I could that as a goal!
Goal for OMI: Keep going, more events!

**Reply by indiebio** [reply]
Get a OMI agent

**Reply by indiebio** [reply]
humbletim: highlight: spent time reflecting, revisiting past projects.
Goal: apply the insights and find incremental ways forward. Make progress by refining known corners and finding dark corners to shine light on them.

**Reply by indiebio** [reply]
Tone: highlights: doing a lot of VRM, asset collections for brands, NFT collections
goals for 2025: continuing with that, better cohesion of standards and things, bring people together from different categories, communication timelines. Replicating functionality to allow open communication.
Goal for OMI: more dogfooding

**Reply by sifr360** [reply]
2025 goals:

To work on/with all of the above.

**Comment by nathangrotticelli** [comment]
Highlights:
- public interactive content (e.g. audio spaces) and traditional marketing
content (e.g. X / LinkedIn posts) to spread awareness for OMI

Members:
- Nathan G LEVITY Race progress

**Comment by mrmetaverse** [comment]
in the spirit of "automating stuff"
 https://github.com/omigroup/omigroup/issues/510
